packages/flutter/lib/src/rendering/view.dart

```diff
@@ -184,6 +204,35 @@ class RenderView extends RenderObject with RenderObjectWithChildMixin<RenderBox>
     }
   }
 
+  void _updateSystemChrome() {
+    if (!automaticSystemUiAdjustment)
+      return;
+    final Rect bounds = paintBounds;
+    final Offset top = new Offset(bounds.center.dx, ui.window.padding.top / ui.window.devicePixelRatio);
+    final SystemUiOverlayStyle upperOverlayStyle = layer.find<SystemUiOverlayStyle>(top);
+    // Only android has a customizable system navigation bar.
```

Comment:
if only android, fuchsia should be in the other branch. :-)

Reply:
Fixed